pom.xml

```diff
@@ -27,11 +27,11 @@
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
 
-    <folio.domain-models-runtime.version>29.3.1</folio.domain-models-runtime.version>
+    <folio.domain-models-runtime.version>30.0.1</folio.domain-models-runtime.version>
     <generate_routing_context>/saml/callback,/saml/regenerate,/saml/login,/saml/check,/saml/configuration
     </generate_routing_context>
 
-    <vertx.version>3.8.5</vertx.version>
+    <vertx.version>3.9.1</vertx.version>
     <pac4j.version>2.0.0</pac4j.version>
     <vertx-pac4j.version>3.0.0</vertx-pac4j.version>
     <jackson.version>2.10.1</jackson.version>
```

Comment:
Consider using [Bill of Materials (Maven BOM)](https://github.com/vert-x3/vertx-stack#bills-of-materials-maven-bom) to replace vertx.version and jackson.version.